src/SignalR/clients/csharp/Http.Connections.Client/src/Internal/WebSocketsTransport.cs

```diff
@@ -224,6 +225,8 @@ private async Task ProcessSocketAsync(WebSocket socket)
             // Wait for send or receive to complete
             var trigger = await Task.WhenAny(receiving, sending).ConfigureAwait(false);
 
+            _stopCts.CancelAfter(_closeTimeout);
```

Comment:
This should be after the cancel pending read

Reply:
I guess it could, doesn't really matter since that's the next line of code that will execute. 

Although, I'm wondering if we even need this here, the only 2 cases we should get here are when StopAsync is called (which triggers the token already), or something server-side/network related kills the websocket in which case close should be able to occur without the cancellation token.

Reply:
Overkill is fine as long as there aren't any ODEs. It'll end up resetting the timer though.

Reply:
Shouldn't ODE, Stop waits on the Running task, which is what this function is, before disposing.